### PR TITLE
Added screenshot contrib

### PR DESCRIPTION
**Pull request by wiedzmin**

### CODE REVIEW

File: util/screenshot/package.lisp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+;;;; package.lisp
+
+(defpackage #:screenshot
+  (:use #:cl :stumpwm :zpng))
+
+(in-package #:screenshot)
+
+(import
```
Comment (jorams) [inline util/screenshot/package.lisp]: This import is unnecessary, all of those symbols are external to the `ZPNG` package and will thus already be `:use`'d by the `defpackage` above.

**Comment by jorams** [comment]
Small improvement, but looks good! I actually looked into taking screenshots like this a few weeks ago, but I'm glad you made it into a contrib.

**Comment by dbjergaard** [comment]
@wiedzmin could you fix this and we'll merge it?

**Comment by wiedzmin** [comment]
Yes. sure.

**Pull request merged**